cypress.env

```diff
@@ -9,6 +9,7 @@ KEYWORKER_API_URL=http://localhost:9191/keyworker/
 TOKENVERIFICATION_API_URL=http://localhost:9191/tokenverification/
 NN_ENDPOINT_URL=http://localhost:20200/
 API_DATA_COMPLIANCE_ENDPOINT_URL=http://localhost:9191/datacompliance/
+API_CLIENT_ID=dev
```

Comment:
This is an interesting one. When we run the integration tests app locally, it picks up what it can from the end file and sort of fills in the rest. It will take any missing values from the nether (or your .env). So if locally you have the API_CLIENT_ID specified, the auth stubs will always work. Circle doesn't have that, so when the tests log out, they try to access an auth stub that doesn't match match of the client id (defaults to elite2apiclient). 

That's my theory anyway :D